test/cli.js

```diff
@@ -141,6 +141,39 @@ describe('command line interface', function() {
           .then(complete)
           .catch(complete);
       });
+
+      it('read input js files', done => {
+        function complete() {
+          should(exitCode).equal(0);
+          should(stdOut.length).above(0);
+          should(stdOut.indexOf('console.log')).above(-1);
+          done();
+        }
+
+        runCmd(cliPath, [`--platform=${platform}`, '--js=test/fixtures/one.js'])
+          .then(complete)
```

Comment:
I think we should be using the 2-arg version of then here and below.

`.then(complete, complete)`

Otherwise the exception thrown by `complete()` when called from `.then(complete)` causes another call to `complete()` from the `.catch(complete)`.